<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CWidgetRunEvo</class>
 <widget class="QWidget" name="CWidgetRunEvo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="label_10">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Actual Chromosome</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_14">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Identification:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>  Generation ID:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLineEdit" name="lineEditGen">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>  Chromosome ID:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLineEdit" name="lineEditChrom">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>  Ancestors Count:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLineEdit" name="lineEditAncestorsCount">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="Line" name="line_2">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_11">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Result:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>  Fitness - max:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QLineEdit" name="lineEditFitMax">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="3">
    <widget class="QLabel" name="label_15">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>  Differention - min:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QLineEdit" name="lineEditDiffMin">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="3">
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>  CA Step no.:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="3">
    <widget class="QLineEdit" name="lineEditFitMaxStep">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="4">
    <widget class="QLabel" name="label_13">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Chromosomes with same result:</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>  Count:</string>
     </property>
    </widget>
   </item>
   <item row="13" column="3">
    <widget class="QLineEdit" name="lineEditSameChromsCount">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="3">
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>  Generation - Avarage:</string>
     </property>
    </widget>
   </item>
   <item row="14" column="3">
    <widget class="QLineEdit" name="lineEditSCGensAvarage">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="15" column="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
